oauth2client/client.py

```diff
@@ -272,7 +272,7 @@ def new_from_json(cls, s):
       An instance of the subclass of Credentials that was serialized with
       to_json().
     """
-    if six.PY3 and isinstance(s, bytes):
```

Comment:
`bytes` is supported in Python 2.6 and newer, `six.binary_type` only makes sense if older versions need to be supported.